src/Orchard.Environment.Extensions.Abstractions/IExtensionPriorityStrategy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Orchard.Environment.Extensions.Features;
+
+namespace Orchard.Environment.Extensions
+{
+    public interface IExtensionPriorityStrategy
+    {
+        double GetPriority(IFeatureInfo feature);
```

Comment:
Ok as you want but for infos currently the `Priority` property of `IFeatureInfo` is a double and there is one strategy which returns `feature.Priority` which is a double ... What do you think?